runtime/onert/core/include/ir/train/LossInfo.h

```diff
@@ -33,11 +33,20 @@ enum class LossReductionType
   Sum,              //< Sum loss reduction type
 };
 
+struct CategoricalCrossentropyParam
+{
+  int32_t axis;
+  float label_smoothing;
+};
+
 struct LossInfo
 {
   LossCode loss_code;
   LossReductionType reduction_type;
-  // TODO Add properties
+  union LossParam {
+    CategoricalCrossentropyParam cce;
+  } loss_param;
```

Comment:
(Optional) How about using polymorphism instead of union? For example, `LossInfo` has a base class and access param info by using it.

Reply:
Okay, I will try to modify it and update it soon. Anyway, I want this pr to go on like this. Thank you.